adr/cli.py

```diff
@@ -156,6 +160,9 @@ def main(args=sys.argv[1:]):
 
     :param list args: command-line arguments.
     """
+    # load config from file
+    config = Configuration(os.path.join(here, '../config.yml'))
```

Comment:
I think this might only work because you specified `python setup.py develop` when installing the repo. I think if you ran `python setup.py install` instead (what happens when adr gets pip installed), it would break. This is because the `config.yml` file won't get copied to the install dir. It works with `develop` because that creates symlinks instead of copying.

I think you'll need to move this into the `adr` module and make sure it's listed in `MANIFEST.in`:
https://docs.python.org/2/distutils/sourcedist.html#manifest